app/routes/groups/group.js

```diff
@@ -99,6 +99,11 @@ router.post('/:slug', access.isAdmin, function(req, res)
 				}
 				case "deleteGroup": 
 				{
+					// Remove reference to group from game
+					Game.removeGroup(null, group._id, function(err)
```

Comment:
Can we instead, make sure the group removal was successful, then do the request? For instance:

```javascript
Game.removeGroup(null, group._id, function(err) {
  if(err) {
    done(err);
    return
  }

  group.remove(function(err) {
    done(err, 'Deleted ' + group.name + ' successfully.'));
    // I don't think you actually need the req.flash since the bottom function in the chain does that anyways
  });
});
```